cmd/webhook/builder.go

```diff
@@ -161,14 +161,14 @@ func (builder CommandBuilder) buildRun() func(*cobra.Command, []string) error {
 			return err
 		}
 
-		dkv1beta2Validator := dynakubev1beta2validation.New(webhookManager.GetAPIReader(), webhookManager.GetConfig())
```

Comment:
Why undo this?
Now only the `v1beta3` Dynakube is validated, which we don't want, we want every version we have to be validated. So our validation logic also versioned.

Reply:
fixed in 38d79f13166e27799292383d0c9c3571f6a58be5